### PR TITLE
Fix #499: FIDO2 Test application: Property to Hide Advanced Option

### LINKED ISSUE #499: FIDO2 Test application: Property to Hide Advanced Option
Add application property to possibly hide advanced settings.

### PR DESCRIPTION
**Pull request by jnpsk**
Added configuration property to be able to hide advanced settings on property configuration level.

### CODE REVIEW

File: powerauth-fido2-tests/src/main/java/com/wultra/security/powerauth/fido2/controller/HomeController.java [hunk 1]
```diff
@@ -48,12 +51,17 @@ public class HomeController {
     private static final String LOGIN_PAGE = "login";
     private static final String PAYMENT_PAGE = "payment";
 
+    @Value("${powerauth.fido2.test.service.hideDeveloperOptions:false}")
+    private Boolean hideDeveloperOption;
```
Comment (banterCZ) [inline powerauth-fido2-tests/src/main/java/com/wultra/security/powerauth/fido2/controller/HomeController.java:55]: Could it be `boolean`?

File: powerauth-fido2-tests/src/main/java/com/wultra/security/powerauth/fido2/controller/HomeController.java [hunk 2]
```diff
@@ -37,7 +40,7 @@
  * @author Jan Pesek, jan.pesek@wultra.com
  */
 @Controller
-@AllArgsConstructor
+@RequiredArgsConstructor
```
Comment (banterCZ) [inline powerauth-fido2-tests/src/main/java/com/wultra/security/powerauth/fido2/controller/HomeController.java:43]: Ok, but later on, we may extract `hideDeveloperOption` and other properties into e.g. `PowerAuthFido2ConfigProperties`.

**Review by banterCZ — APPROVED** [review]
Looks good.

**Review by zcgandcomp — APPROVED** [review]
Looks OK.

**Pull request merged**